.github/workflows/ci.yaml

```diff
@@ -50,6 +50,23 @@ jobs:
           mix deps.compile --warnings-as-errors
           mix dialyzer --plt
 
+  tlint:
+    name: Run TLint over checks
+    runs-on: ubuntu-20.04
+    container:
+      image: ghcr.io/trento-project/tlint:latest
+      volumes:
+        - ${{ github.workspace }}:/data
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Run TLint
+        run: "/home/tlint/tlint lint -f /data/priv/catalog"
+      
+
```

Comment:
I would remove one of the newlines by the way. 1 is enough XD

Reply:
ah yes the infamous newline

I've removed them from the whole file since the CI was really hard to read by using it in every step